CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0 - 2024/01/17
+* 🚀 [BREAKING] Allow transformers to be included across views. See [this](https://github.com/procore-oss/blueprinter#transform-across-views) section of the README for details, also PR [#372](https://github.com/procore-oss/blueprinter/pull/372) and issue [#225](https://github.com/procore-oss/blueprinter/issues/225). Note this changes the behavior of transformers which were previously only applied to the view they were defined on. Thanks to [@njbbaer](https://github.com/njbbaer) and [@bhooshiek-narendiran](https://github.com/bhooshiek-narendiran).
+* 💅 [ENHANCEMENT] Add reflection on views, fields, and associations. See PRs [#357](https://github.com/procore-oss/blueprinter/pull/357), [#358](https://github.com/procore-oss/blueprinter/pull/358), and issue [#341](https://github.com/procore-oss/blueprinter/issues/341). Thanks to [@jhollinger](https://github.com/jhollinger).
```

Comment:
I think the introduction of an extensions API is worth calling out explicitly as a separate enhancement!

Reply:
Updated!